pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -16,6 +16,33 @@
 #@   end
```

Comment:
Rather than recompiling `tanzu cli` on changes, should be able to invoke the ytt directly with something like this (provided via slack discussion for this work):

```bash
ytt -f providers/yttcb -f providers/infrastructure-vsphere/v1.3.1/yttcc -f providers/infrastructure-vsphere/yttcc -f providers/yttcc -f bom -f providers/config_default.yaml -f /some/final/config/for/workload-ytt.yaml | less
```
using a pretty basic config:
```
# workload-ytt.yaml
# will have to provide additional vars for pinniped.
CLUSTER_NAME: mycluster
CLUSTER_PLAN: dev
INFRASTRUCTURE_PROVIDER: vsphere
OS_ARCH: amd64
OS_NAME: ubuntu
OS_VERSION: '20.04'
```